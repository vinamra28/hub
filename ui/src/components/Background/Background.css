.hub-background-image {
  background-image: url('../../assets/logo/background.jpg');
  background-repeat: no-repeat;
  background-attachment: fixed;
  background-size: cover;
  padding: 2em;
}

.hub-background-heading {
  color: white;
  text-align: center;
}

.hub-tag-line {
  color: white;
  text-align: center;
}
