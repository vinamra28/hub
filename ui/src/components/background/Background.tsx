import { Text, TextContent, TextVariants } from '@patternfly/react-core';
import '@patternfly/react-core/dist/styles/base.css';
import React from 'react';
import './background.css';

const BackgroundImageHeader: React.FC = () => (
  <div className="background-image">
    <TextContent>
      <Text component={TextVariants.h1} className="heading">
        Welcome to Tekton Hub
      </Text>
      <Text component={TextVariants.h3} className="tag-line">
        Discover, search and share reusable Tasks and Pipelines
      </Text>
    </TextContent>
  </div>
);

export default BackgroundImageHeader;
