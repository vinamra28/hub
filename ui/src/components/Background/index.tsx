import { Text, TextContent, TextVariants } from '@patternfly/react-core';
import '@patternfly/react-core/dist/styles/base.css';
import React from 'react';
import './Background.css';

const BackgroundImageHeader: React.FC = () => (
  <div className="hub-background-image">
    <TextContent>
      <Text component={TextVariants.h1} className="hub-background-heading">
        Welcome to Tekton Hub
      </Text>
      <Text component={TextVariants.h3} className="hub-tag-line">
        Discover, search and share reusable Tasks and Pipelines
      </Text>
    </TextContent>
  </div>
);

export default BackgroundImageHeader;
