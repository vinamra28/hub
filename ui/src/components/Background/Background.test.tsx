import React from 'react';
import renderer from 'react-test-renderer';
import Background from '.';

it('should render the background component', () => {
  const component = renderer.create(<Background />).toJSON();
  expect(component).toMatchSnapshot();
});
