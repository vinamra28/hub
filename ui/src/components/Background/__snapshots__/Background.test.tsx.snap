// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the background component 1`] = `
<div
  className="hub-background-image"
>
  <div
    className="pf-c-content"
  >
    <h1
      className="hub-background-heading"
      data-pf-content={true}
    >
      Welcome to Tekton Hub
    </h1>
    <h3
      className="hub-tag-line"
      data-pf-content={true}
    >
      Discover, search and share reusable Tasks and Pipelines
    </h3>
  </div>
</div>
`;
